docs/foundations/color/context.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: layout-basic.njk
+title: Color Context
+order: 1
+tags:
+  - foundations
+---
+<script type="module">
+  import '@rhds/elements/rh-blockquote/rh-blockquote.js';
+  import '@rhds/elements/rh-cta/rh-cta.js';
+  import '@rhds/elements/rh-card/rh-card.js';
+  import '@rhds/elements/rh-tag/rh-tag.js';
+  import '@rhds/elements/lib/elements/rh-context-demo/rh-context-demo.js';
+</script>
+
+Red Hat Design System comes with a contextual colour-theming feature called 
+"Color Context", designed to make page developers' and content authors' jobs
+easier and to improve customers' digital experiences. Authors and developers who
+adopt the color context system will produce accessible, branded experiences with 
+less effort and greater cross-property consistency.
+
+<rh-blockquote>
+  HTML design systems help teams ship better digital experiences, faster.
+</rh-blockquote>
+
+## How Color Context works
+<abbr title="red hat design system">RHDS</abbr>' color context system is an 
+*HTML* and *CSS* system with some supporting JavaScript[1].
+The color context system has two main parts: **providers** and **consumers**. 
+Providers actively define a color theme, while consumers passively accept their 
+theme from the nearest provider ancestor.
+
+### Color context providers
+Context **providers** define a **color palette** for themselves and their child 
+elements. There are currently six main color palettes:
+1. lightest
+1. lighter
+1. light
+1. dark
+1. darker
+1. darkest
+
+Context providers typically use the `lightest` color palette as the default. 
+Authors may define the color palette of a container using the `color-palette` HTML
+attribute. So for example, to create a card with the darkest color palette, use 
+this HTML:
+
+```html
```

Comment:
What do you think about adding `<rh-card>` examples next to the code here? It could look something like this to match other docs pages. I think using a screenshot of the card is fine, but if we could use the actual element, that would be better. ([Figma link](https://www.figma.com/file/jm5tyGBHqpZSkagKSxGTqh/color-context-example?type=design&node-id=0%3A1&mode=design&t=rA1NoYKFmGQEw3KE-1))
<img width="476" alt="Screenshot 2023-11-04 at 12 38 21 PM" src="https://github.com/RedHat-UX/red-hat-design-system/assets/95588923/50b19fd8-d049-407e-b5be-c6a3940527d3">